### PR TITLE
[Fix] fix triton server chatbot init error

### PR DESCRIPTION
**Pull request by AllentDan**
@zhulinJulia24 May have a check.

### CODE REVIEW

File: lmdeploy/serve/turbomind/chatbot.py [hunk 1]
```diff
@@ -68,6 +69,7 @@ def __init__(self,
                  ignore_eos: bool = False,
                  log_level: int = logging.INFO,
                  display: bool = False,
+                 gen_cfg: GenerationConfig = GenerationConfig(),
```
Comment (ispobock) [inline lmdeploy/serve/turbomind/chatbot.py:72]: 1. The initialization in https://github.com/InternLM/lmdeploy/blob/920d7196336f437e6383f39c04804236ef5270da/benchmark/profile_serving.py#L84 also should be fixed.
2. Why don't put `ignore_eos` to GenerationConfig?

Comment (AllentDan) [reply lmdeploy/serve/turbomind/chatbot.py]: `GenerationConfig` is not supposed to be used in the init function. I just chose a quick while not elegant way to fix the bug.

Comment (ispobock) [reply lmdeploy/serve/turbomind/chatbot.py]: Is this parameter deprecated? Maybe we can remove it. https://github.com/InternLM/lmdeploy/blob/e7b2833441be6a4ac374ec191001ba6ae97647e0/benchmark/profile_serving.py#L86

**Review by zhulinJulia24 — APPROVED** [review]
lgtm

**Comment by zhulinJulia24** [comment]
fixed  https://github.com/zhulinJulia24/lmdeploy/actions/runs/8265142135/job/22612492797

**Review by zhulinJulia24 — APPROVED** [review]
LGTM

**Review by ispobock — APPROVED** [review]
LGTM

**Review by lvhan028 — APPROVED** [review]

**Pull request merged**